<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulated annealing </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="updateButton">
          <property name="toolTip">
           <string>Start annealing simulation</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
          <property name="text">
           <string>&amp;Simulate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="toolTip">
           <string>Reset values and resize grid</string>
          </property>
          <property name="text">
           <string>&amp;Reset (resize)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Si&amp;ze:</string>
          </property>
          <property name="buddy">
           <cstring>widthText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="widthText"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="heightText"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Energy:</string>
          </property>
          <property name="buddy">
           <cstring>energyText</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="temperatureText"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="tRatioText"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Iterations:</string>
          </property>
          <property name="buddy">
           <cstring>iterationsText</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="iterationsText"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Temperature ratio:</string>
          </property>
          <property name="buddy">
           <cstring>tRatioText</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Temperature:</string>
          </property>
          <property name="buddy">
           <cstring>temperatureText</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="energyText"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Form" name="form" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Form</class>
   <extends>QWidget</extends>
   <header>form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>updateButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>widthText</tabstop>
  <tabstop>heightText</tabstop>
  <tabstop>tRatioText</tabstop>
  <tabstop>temperatureText</tabstop>
  <tabstop>energyText</tabstop>
  <tabstop>iterationsText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
